app/src/pages/header/components/AddMovieForm.jsx

```diff
@@ -18,25 +20,40 @@ const ColoredButtonWrapper = styled(ColoredButton)`
 `
 
 function AddMovieForm(props) {
-    const [genres, setGenres] = useState([]);
-    
-    useEffect(() => {
-        Promise
-            .resolve(genresFromJson)
-            .then(genres => setGenres(genres))
-    }, [])
+    const [title, setTitle] = useState('');
+    const [releaseDate, setReleaseDate] = useState('');
+    const [url, setUrl] = useState('');
+    const [genres, setGenres] = useState('');
+    const [overview, setOverview] = useState('');
+    const [runtime, setRuntime] = useState(0);
+
+    function confirmAdd(event){
+        event.preventDefault();
+        var movie = {

```

Comment:
Do we need to use 'var'? It is obsolete keyword

Reply:
+ how about usage of the Model?

Reply:
Done: replaced `var` to `let`, added `Movie` model